### PR TITLE
Added root calculator #41

### PR DESCRIPTION
**Pull request by sharmavikas4**
Added the calculator that can find roots of the quadratic equation. Here are the glimpse of it.
Screenshot (63)
Screenshot (64)
Screenshot (65)

### CODE REVIEW

File: calculators/root-calculator/script.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        const r1 = ((-b + Math.sqrt(d))/(2*a)).toFixed(2);
+        const r2 = ((-b - Math.sqrt(d))/(2*a)).toFixed(2);
+        ans.innerHTML = "The roots are real and distinct : " + r1 + " & " + r2; 
+    }
+    else if (d==0){
+        const r = ((-b)/(2*a)).toFixed(2);
+        ans.innerHTML = "The roots are real and equal : "  + r + " & " + r;
+    }
+    else if (d<0){
+        ans.innerHTML = "The roots doesn't exist";
```
Comment (RuanAragao) [inline calculators/root-calculator/script.js:24]: Just a grammar correction: "The roots don't exist".

**Comment by sharmavikas4** [comment]
Corrected the grammatical error

File: calculators/root-calculator/script.js (hunk 1 above)
Comment (sharmavikas4) [reply calculators/root-calculator/script.js]: Corrected the grammatical mistake

**Comment by sharmavikas4** [comment]
Please merge it now @RuanAragao I have corrected the grammatical mistake. After merging please put hacktoberfest accepted label.

**Pull request closed without merging**